grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -259,6 +260,10 @@ class Order implements Serializable {
         return activeOrderItems?.every { OrderItem orderItem -> orderItem.isCompletelyFulfilled() }
     }
 
+    Boolean isPartiallyShipped() {
+        return activeOrderItems?.any { OrderItem orderItem -> orderItem.isCompletelyFulfilled() }
```

Comment:
@drodzewicz what if every item is completely fulfilled? Then this will be both `shipped` and `partiallyShipped`. I think you should either check if it is `&& !shipped` or check if `activeOrderItems?.any { it.isPartiallyFulfilled() }` (I think I prefer the later version)

Reply:
I prefer using a static type with closures but we can discuss if this is preferred.